benchexec/test_benchexec_integration.py

```diff
@@ -27,11 +27,15 @@
 import tempfile
 import unittest
 import zipfile
+
+from xml.etree import ElementTree
+
 sys.dont_write_bytecode = True # prevent creation of .pyc files
 
 here = os.path.dirname(__file__)
 base_dir = os.path.join(here, '..')
 bin_dir = os.path.join(base_dir, 'bin')
+tmp_dir = os.path.join(here, 'test_integration')
```

Comment:
This is one of the things I don't like ;-)
The alternative to this is having an 'expected file' that has wrong files-references for the directory it is in, but correct references if it were in the temporary directory.

Since this makes the 'expected file' basically a bad example, I preferred this way, but I understand your point.
So should I change this?